test/t8_test_descendant.cxx

```diff
@@ -0,0 +1,160 @@
+/*
```

Comment:
This test does not finish.
Either it just takes too long or it runs into an infinite loop.

I deactivated it on my branch for the time being. Please investigate and reactivate if fixed.

Reply:
It just takes a lot of time to finish this test.